CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 ## Next Version
 
+* Change the `replaces` attribute from a single class to an array. This gives the API more flexibility and avoids redundant classes. E.g. one Dagger module with several binding and provider methods may wish to replace multiple other Dagger modules:
+* Introduce the new `@ContributesBinding` annotation. This annotation allows you to contribute binding methods without actually writing a Dagger module.
+```
+interface Authenticator
+
+@ContributesBinding(AppScope::class)
+class RealAuthenticator @Inject constructor() : Authenticator
+
+// The generated and automatically included Dagger module would look similar like this:
+@Module
+@ContributesTo(AppScope::class)
+abstract class AuthenticatorModule {
+  @Binds abstract fun bindRealAuthenticator(authenticator: RealAuthenticator): Authenticator
+}
+```
+* Support nested classes for contributed Dagger modules and component interfaces if the outer class uses a different scope #45.
```

Comment:
Did you want to link to PRs for the other items in the list as well?

Reply:
There were no external issues for them. I only link to issues and no PRs usually.